### ISSUE TITLE
Address java.lang.RuntimeException when searching for songs

### ISSUE THREAD

**Issue by aaronrai24** [aaronrai24/DollarDiscordBot#60]
Bug report: Somehow a numberformatexception is getting thrown. Is this detrmiental, does this always get thrown when dollar is unable to find a song?

Submitted by: mfcash
Server: mfDiscord

**Comment by aaronrai24** [comment]
stack trace:
```
2023-06-29 21:20:29.577 ERROR 2449 --- [lava-daemon-pool-info-loader-2-thread-1] lavalink.server.player.AudioLoader       : Load failed

com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something went wrong when looking up the track
        at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:44) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.dispatchItemLoadFailure(DefaultAudioPlayerManager.java:242) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:231) ~[lavaplayer-1.4.0.jar!/:na]
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
        at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
        Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation: 
  lavaplayer.version: 1.4.0-original
  os.arch: aarch64
  os.name: Linux
  os.version: 5.19.17-meson64
  java.vendor: Ubuntu
  java.version: 11.0.19
  java.runtime.version: 11.0.19+7-post-Ubuntu-0ubuntu122.04.1
  java.vm.version: 11.0.19+7-post-Ubuntu-0ubuntu122.04.1
Caused by: java.lang.RuntimeException: java.lang.NumberFormatException: For input string: "98M plays"
        at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:58) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchMusicProvider.loadSearchMusicResult(YoutubeSearchMusicProvider.java:71) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.searchMusic(YoutubeAudioSourceManager.java:293) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.searchMusic(YoutubeAudioSourceManager.java:248) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.route(DefaultYoutubeLinkRouter.java:39) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:216) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:147) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:449) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:431) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:223) ~[lavaplayer-1.4.0.jar!/:na]
        ... 4 common frames omitted
Caused by: java.lang.NumberFormatException: For input string: "98M plays"
        at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:65) ~[na:na]
        at java.base/java.lang.Integer.parseInt(Integer.java:652) ~[na:na]
        at java.base/java.lang.Integer.parseInt(Integer.java:770) ~[na:na]
        at com.sedmelluq.discord.lavaplayer.tools.DataFormatTools.durationTextToMillis(DataFormatTools.java:152) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchMusicProvider.extractMusicTrack(YoutubeSearchMusicProvider.java:157) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchMusicProvider.lambda$extractMusicSearchPage$0(YoutubeSearchMusicProvider.java:119) ~[lavaplayer-1.4.0.jar!/:na]
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541) ~[na:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchMusicProvider.extractMusicSearchPage(YoutubeSearchMusicProvider.java:118) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchMusicProvider.extractSearchResults(YoutubeSearchMusicProvider.java:80) ~[lavaplayer-1.4.0.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchMusicProvider.loadSearchMusicResult(YoutubeSearchMusicProvider.java:68) ~[lavaplayer-1.4.0.jar!/:na]
        ... 12 common frames omitted
```

**Comment by aaronrai24** [comment]
Could very well be a java versioning issue, should upgrade Java to Java 17 as required by Lavalink fork and then monitor the logs again.

**Comment by aaronrai24** [comment]
Will be resolved by #91 (lavalink update)

**Comment by aaronrai24** [comment]
Fixed with #92
